omegatrade-app/backend/app/models/company.model.js

```diff
@@ -0,0 +1,61 @@
+'user strict';
+const database = require('./../config/database.js');
+var Company = function () { };
+
+Company.getAll = async function (cb) {
+    try {
+        const query = { sql: 'select companyId , companyName , companyShortCode , created_at from companies', json: true, };
+        const [companies] = await database.run(query);
```

Comment:
no need to assign query object into query as it can be used directly.

Reply:
this issue has been resolved.